api/models/user/user_model.py

```diff
@@ -59,10 +59,16 @@ class User(AbstractUser):
     """Custom Corona Testing  user implementation
     """
 
-    email = models.EmailField(unique=True)
-    username = models.CharField(max_length=25, null=True, blank=True)
-    age = models.IntegerField(null=True, blank=True)
+    # email = models.EmailField(unique=True)
+    
+    # age = models.IntegerField(null=True, blank=True)
+
+    username = models.CharField(max_length=25, null=True, blank=True, unique=False)
+    # max length of SHA512 is 128 chars
+    phone_number_hash = models.CharField(max_length=150, unique=True)
```

Comment:
FYI this will cause an error with existing users since there is no default, I will fix this since it's originating at a different level

Reply:
I didn't know there are already existing users